### PR TITLE
README for BigDL-LLM on docker

### PR DESCRIPTION
**Pull request by Zhengjin-Wang**
* add instruction for MacOS/Linux
* modify html label of gif images
* organize new structure

### CODE REVIEW

**Review by Oscilloscope98 — COMMENTED** [review]
The link part LGTM

**Comment by glorysdj** [comment]
Getting started with BigDL LLM
1. Getting started with BigDL LLM on Windows
2. Getting started with BigDL LLM on Linux/MacOS
3. Getting started with BigDL LLM on XPU
4. BigDL LLM Serving on CPU
5. BigDL LLM Serving on XPU
6. BigDL LLM Fine Tuning on CPU
7. BigDL LLM Fine Tuning on XPU

File: docker/llm/README.md [hunk 1]
```diff
@@ -1,29 +1,43 @@
-## Getting started with BigDL LLM on Windows
+## Getting started with BigDL LLM on cpu
```
Comment (jason-dai) [inline docker/llm/README.md:1]: The title is confusing; should be something like "Docker user guide for BigDL-LLM on CPU"

**Comment by jason-dai** [comment]
> Getting started with BigDL-LLM
> 
> 1. Docker user guide for BigDL-LLM on CPU
>    1.1. BigDL-LLM on Windows
>    1.2. BigDL-LLM on Linux/MacOS
> 2. Docker user guide for BigDL LLM on XPU
> 3. BigDL-LLM Serving on CPU
> 4. BigDL-LLM Serving on XPU
> 5. BigDL-LLM Fine Tuning on CPU
> 6. BigDL-LLM Fine Tuning on XPU

Need to make it clear in all the titles/subtitles that it is for using BigDL-LLM in docker container, otherwise it can be confusing to the users

**Review by glorysdj — APPROVED** [review]
LGTM

**Comment by glorysdj** [comment]
> > Getting started with BigDL-LLM
> > 
> > 1. Docker user guide for BigDL-LLM on CPU
> >    1.1. BigDL-LLM on Windows
> >    1.2. BigDL-LLM on Linux/MacOS
> > 2. Docker user guide for BigDL LLM on XPU
> > 3. BigDL-LLM Serving on CPU
> > 4. BigDL-LLM Serving on XPU
> > 5. BigDL-LLM Fine Tuning on CPU
> > 6. BigDL-LLM Fine Tuning on XPU
> 
> Need to make it clear in all the titles/subtitles that it is for using BigDL-LLM in docker container, otherwise it can be confusing to the users

OK, We will change title to Getting started with BigDL-LLM in Docker.

Getting started with BigDL-LLM in Docker

Docker user guide for BigDL-LLM on CPU
1.1. BigDL-LLM on Windows
1.2. BigDL-LLM on Linux/MacOS
Docker user guide for BigDL LLM on XPU
Docker user guide for BigDL-LLM Serving on CPU
Docker user guide for BigDL-LLM Serving on XPU
Docker user guide for BigDL-LLM Fine Tuning on CPU
Docker user guide for BigDL-LLM Fine Tuning on XPU

**Comment by jason-dai** [comment]
> Docker user guide for BigDL-LLM on CPU 

Is it for installation?

**Comment by glorysdj** [comment]
> > Docker user guide for BigDL-LLM on CPU
> 
> Is it for installation?
It is for docker installation on Windows(a quick reference) for newbies, container launch, chat, and tutorial development.

**Comment by jason-dai** [comment]
> > > Docker user guide for BigDL-LLM on CPU
> > 
> > 
> > Is it for installation?
> 
> It is for docker installation on Windows(a quick reference) for newbies, container launch, chat, and tutorial development.

User guide is to general; maybe installation guide?

**Comment by glorysdj** [comment]
> > > > Docker user guide for BigDL-LLM on CPU
> > > 
> > > 
> > > Is it for installation?
> > 
> > 
> > It is for docker installation on Windows(a quick reference) for newbies, container launch, chat, and tutorial development.
> 
> User guide is to general; maybe installation guide?

OK, get it.

**Pull request merged**